### PR TITLE
Updated Code

### PR DESCRIPTION
**Pull request by aman-netizen-ux**
Updated Code of Company Details Screen. (Fixed a minor issue)

### CODE REVIEW

**Review by Anik-Chakraborty — APPROVED** [review]

File: lib/Web/screens/company_details.dart [hunk 1]
```diff
@@ -17,270 +33,391 @@ class CompanyDetails extends StatefulWidget {
...
+                                          shape: RoundedRectangleBorder(
+                                            borderRadius:
+                                                BorderRadius.circular(5),
+                                          ),
+                                          backgroundColor:
+                                              const Color(0xFF000066),
+                                          fixedSize: Size(33.w, 7.h),
+                                        ),
+                                        onPressed: () async {
+                                          // if (_formKey.currentState!.validate()) {
```
Comment (chirag4601) [inline lib/Web/screens/company_details.dart:302]: remove such comment if not useful

File: lib/Web/screens/login.dart [hunk 2]
```diff
@@ -32,300 +43,562 @@ class _LoginWebState extends State<LoginWeb> {
...
+                                    child: Container(
+                                      color: white,
+                                      child: TextFormField(
+                                        autofocus: true,
+                                        autofillHints: autofillHints,
+                                        decoration: InputDecoration(
+                                          hintStyle: TextStyle(
+                                              decoration:
+                                                  TextDecoration.underline,
+                                              decorationColor: Color.fromARGB(
```
Comment (chirag4601) [inline lib/Web/screens/login.dart:252]: Always use colours from colors.dart file

File: lib/Web/screens/login.dart [hunk 3]
```diff
@@ -2,13 +2,24 @@ import 'dart:developer';
 import 'package:flutter/foundation.dart' show kIsWeb;
```
Comment (chirag4601) [inline lib/Web/screens/login.dart:2]: try reducing the length of this and the company_details.dart file by separating static widgets and making a separate file for them in the widgets folder

File: lib/Web/screens/company_details.dart [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
 import 'package:shipper_app/Web/screens/home_web.dart';
+import 'package:shipper_app/constants/colors.dart';
 import 'package:shipper_app/functions/alert_dialog.dart';
+
+import '../../constants/colors.dart';
```
Comment (chirag4601) [inline lib/Web/screens/company_details.dart:13]: when you import files, there are always 2 options for import name, one starting with package:shipper_app.. and the other with ../../ , always choose the one with package name - its easier to shift files/ rename files

**Pull request merged**